Prepare preflight for border compatibility

RobinMalfait opened the pull request:
This PR prepares the `preflight.css` so that we can introduce border style compatibility in a future PR.

RobinMalfait commented:
* **#14746** <a href="https://app.graphite.dev/github/pr/tailwindlabs/tailwindcss/14746?utm_source=stack-comment-icon" target="_blank"><img src="https://static.graphite.dev/graphite-32x32-black.png" alt="Graphite" width="10px" height="10px"/></a>
* **#14745** <a href="https://app.graphite.dev/github/pr/tailwindlabs/tailwindcss/14745?utm_source=stack-comment-icon" target="_blank"><img src="https://static.graphite.dev/graphite-32x32-black.png" alt="Graphite" width="10px" height="10px"/></a> 👈
* `next`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @RobinMalfait and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment"><img src="https://static.graphite.dev/graphite-32x32-black.png" alt="Graphite" width="11px" height="11px"/> <b>Graphite</b></a>
  <!-- Current dependencies on/for this PR: -->

philipp-spiess commented:
### Merge activity

* **Oct 22, 12:07 PM EDT**: The merge label 'merge-queue' was detected. This PR will be added to the [Graphite merge queue](https://app.graphite.dev/merges?org=tailwindlabs&repo=tailwindcss) once it meets the requirements.

RobinMalfait committed:
Prepare preflight for border compatibility (#14745)

This PR prepares the `preflight.css` so that we can introduce border style compatibility in a future PR.
